src/__Plugin.ino

```diff
@@ -1196,6 +1202,10 @@ byte PluginCall(byte Function, struct EventStruct *event, String& str)
           if (Plugin_id[x] != 0) {
             if (Plugin_ptr[x](Function, event, str)) {
               delay(0); // SMY: call delay(0) unconditionally
+#ifdef USES_C014
+              // and here too?
+              CPlugin_014_Feedback(Function, &TempEvent, str);
```

Comment:
What is this comment about? What does it refer to?

Reply:
I was asking myself why `Plugin_ptr[x](Function, event, str)` is called twice.
As CPluginCall(CPLUGIN_ACKNOWLEDGE, event, str); is now implemented I removed this.